.github/workflows/release_flow.yml

```diff
@@ -81,6 +81,10 @@ jobs:
       run: pip install pyinstaller
     - name: Build with pyinstaller for ${{matrix.TARGET}}
       run: ${{matrix.CMD_BUILD}}
+    - name: Upload a Build Artifact
```

Comment:
I think this is redundant here since all artifacts generated as made available as a release asset.
Can you please remove this?

But it was a good idea. I think we'll use this in some sort of incremental release. That will be a different context than this workflow tho